src/main/java/codeu/model/data/Message.java

```diff
@@ -16,6 +16,9 @@
 
 import java.time.Instant;
 import java.util.UUID;
+import org.commonmark.node.*;
```

Comment:
Please avoid * imports

Reply:
I will try to find a better way to implement styles without * imports. Would you recommend finding a new API, finding the libraries within org.commonmark.node that are actually used, or a different approach?

Reply:
* Just means import all, I'm telling you to figure out the specific item you need to import it and change it to that.

I.e.

`import java.util.UUID;`
vs.
`import java.util.*;`